versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx

```diff
@@ -51,14 +58,32 @@ We see in the graph above that there are two usage metrics that LangSmith charge
 The first metric tracks all traces that you send to LangSmith. The second tracks all traces that also have our Extended 400 Day Data Retention.
 For more details, see our [data retention conceptual docs](../../concepts/usage_and_billing/data_retention_billing).
 
+LangSmith Traces usage is measured per workspace. This is because workspaces often represent development environments (as in our example),
+or teams within an organization, and as a LangSmith administrator, we want to know which environment or team is responsible for
+the majority of our costs.
+
+:::note
+LangSmith's Usage Graph and Invoice use the term `tenant_id` to refer to a workspace ID. They are interchangeable.
+:::
+
+In the above image, the vast majority of usage is in the workspace with ID `c27dd32c-7c80-4e8c-acde-bfcb67a90ab2`. We can
+go to `Settings` -> `Workspaces`, and hover our mouse over the `Workspace ID` button to find the one with a matching ID:
+
+![](./static/p1findworkspaceid_v2.png)
+
 ### Invoices
 
-We head to the `Invoices` tab to see how our usage translates into spend in the current billing period.
+We understand what usage looks like in terms of traces, but need to now translate that into spend. To do so,
```

Comment:
with this type of documentation, I almost think we should combine usage graph and invoices so it's a comprehensive view of spend